### DISCUSSION TITLE
Discussions are now open!

### DISCUSSION THREAD

**Discussion by babaktaremi** [babaktaremi/Clean-Architecture-Template#7] [Announcements]
## 👋 Welcome!
from now on you can use discussion page for project improvement ! We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded.
  * Be respectful toward others and try to back up your claims with facts , test results and reliable sources.
  *  Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RohollahHosseini** [comment]
very good ✌✌✌
